waspc/src/Wasp/Lib.hs

```diff
@@ -173,3 +180,16 @@ compileAndRenderDockerfile waspDir compileOptions = do
     Right appSpec -> do
       dockerfileOrGeneratorErrors <- DockerGenerator.compileAndRenderDockerfile appSpec
       return $ left (map show . toList) dockerfileOrGeneratorErrors
+
+deploy :: FilePath -> Path' Abs (Dir WaspProjectDir) -> [String] -> IO ()
```

Comment:
I wonder if this should go here, or maybe be in CLI somewhere.

Lib here is kind of an interface toward compiler.

This deploy part -> it kind of deals a bit more with external stuff? Not sure hm. Ok, I guess it can be here.

Reply:
We would have to export a lot of stuff across to CLI in that case, like `runNodeCommandAsJob`. Mainly put it here to keep those out.

Reply:
Ok makes sense then.